### DISCUSSION TITLE
How to get started with Zigbee

### DISCUSSION THREAD

**Discussion by RutgerBeyen** [seaky/nspanel_pro_tools_apk#37] [Q&A]
Looking for some help on how to get started with zigbee on the NsPanel pro.
I got the device running the HomeAssistant companion app and using it as a wall panel dashboard.
How can I now use the zigbee module (gateway is fine - I'm not using zigbee at all currently), pair some devices and have the status reported back/integrated into HomeAssistant ?
My NSPanelPro is on v2.1.0
Many thanks!

**Comment by seaky** [comment]
hi

a tons of info here: 
https://github.com/seaky/nspanel_pro_roottool_apk/issues/3
